crypto/context.c

```diff
@@ -299,127 +514,114 @@ int ossl_lib_ctx_is_global_default(OSSL_LIB_CTX *ctx)
     return 0;
 }
 
-static void ossl_lib_ctx_generic_new(void *parent_ign, void *ptr_ign,
-                                     CRYPTO_EX_DATA *ad, int index,
-                                     long argl_ign, void *argp)
+void *ossl_lib_ctx_get_data(OSSL_LIB_CTX *ctx, int index,
+                            const OSSL_LIB_CTX_METHOD *meth)
 {
-    const OSSL_LIB_CTX_METHOD *meth = argp;
-    OSSL_LIB_CTX *ctx = ossl_crypto_ex_data_get_ossl_lib_ctx(ad);
-    void *ptr = meth->new_func(ctx);
+    void *p;
+
+    ctx = ossl_lib_ctx_get_concrete(ctx);
+    if (ctx == NULL)
+        return NULL;
+
+    switch (index) {
+        case OSSL_LIB_CTX_PROPERTY_STRING_INDEX:
```

Comment:
We don't indent the `case` bit and only indent the body once.

qv https://github.com/openssl/technical-policies/blob/master/policies/coding-style.md#chapter-3-placing-braces-and-spaces